docs/how-to-guides/themes/theme-json.md

```diff
@@ -595,7 +595,7 @@ The Gutenberg plugin extends the styles available from WordPress 5.8, so they ca
 The tabs below show WordPress 5.8 supported styles and the ones supported by the Gutenberg plugin.
 </div>
 
-Each block declares which style properties it exposes via the [block supports mechanism](../block-api/block-supports.md). The support declarations are used to automatically generate the UI controls for the block in the editor. Themes can use any style property via the `theme.json` for any block ― it's the theme's responsibility to verify that it works properly according to the block markup, etc.
+Each block declares which style properties it exposes via the [block supports mechanism](../../reference-guides/block-api/block-supports.md). The support declarations are used to automatically generate the UI controls for the block in the editor. Themes can use any style property via the `theme.json` for any block ― it's the theme's responsibility to verify that it works properly according to the block markup, etc.
```

Comment:
I think this is the URL structure used in documentations - `/docs/reference-guides/block-api/block-supports.md`.

Everything else looks good 👍

Reply:
I have made the change you suggested. Please take a look at it and let me know if this needs any changes.
Thanks

Reply:
Thanks for the quick update!